src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -24,4 +24,6 @@ public class VisionConstants {
           Units.inchesToMeters(-5.5),
           Units.inchesToMeters(12.0),
           new Rotation3d(0, 0, 0));
+
+  public final Integer safetyThreshold = 10;
```

Comment:
Could we rename this something like "minimumVisionMeasurementInitCount" or similar to indicate that this is a counter of how many vision measurements we need to trust that the robot is in the correct location?